src/main/java/org/icgc/argo/program_service/converter/Grpc2JsonConverter.java

```diff
@@ -49,7 +52,30 @@ public String getJsonFromObject(Object response) {
       throw new ProgramRuntimeException(e.getMessage());
     }
   }
-
+  public UpdateProgramRequest prepareUpdateProgramRequest (UpdateProgramRequestDTO updateProgramRequestDTO) {

```

Comment:
So this looks like we're mixing up our GRPC messages (UpdateProgramRequest) with our DTOs from the HTTP endpoints. I mention in the comment on the `updateProgramWithDataCenter` function that it is best not to involve the GRPC objects in our service logic... If so, then hopefully this whole function can be skipped.